### DISCUSSION TITLE
zustand website animation

### DISCUSSION THREAD

**Discussion by speedweed-ctrl** [pmndrs/zustand#1011] [General]
out of place question but does anyone knows how they did the back round animation for their website ( https://zustand.surge.sh/ )

**Comment by dai-shi** [comment]
You might want to ask in the poimandres discord server: https://discord.gg/poimandres
(No guarantee if anyone can answer there.)

The latest demo is here: https://zustand-demo.pmnd.rs
The code is here: https://github.com/pmndrs/zustand/tree/main/examples
